package.json

```diff
@@ -15,7 +15,7 @@
     "test:links:internal": "run-p -r start check-links:internal",
     "test:links:external": "run-p -r start check-links:external",
     "check-links:internal": "wait-on tcp:8080 && blc http://localhost:8080 --recursive --exclude-external --ordered",
-    "check-links:external": "wait-on tcp:8080 && blc http://localhost:8080 --exclude 'http://localhost:8000/dev/' --exclude 'https://github.com/Polymer/internal*' --exclude 'https://github.com/PolymerLabs/lit.dev*' --exclude 'https://www.npmjs.com/*' --get --recursive --host-requests 1"
+    "check-links:external": "wait-on tcp:8080 && blc http://localhost:8080 --exclude 'http://localhost:8000/dev/' --exclude 'https://www.npmjs.com/*' --exclude 'http://localhost:8000/slack-invite/' --get --recursive --exlude-internal --host-requests 1"
```

Comment:
Sorry if I missed this from the PR description, but still not sure I understand when we need the exclusions, or why.  E.g. why is npm here but not mdn, which we also link to a lot? It seems like a reasonable default behavior for the link checker would be to follow external links, but not any transitive links? Also what does the `--host-requests` flag do? I couldn't find it mentioned at https://github.com/stevenvachon/broken-link-checker.

Reply:
Also just noticed there's a typo: `--exlude-internal`

Reply:
Sorry for the rambling PR description.
The typo you noticed hints to the "fix" I ended up using. It tricked me.

The --host-requests flag is documented in the `--help` text as: `Concurrent requests limit per host.`.
We need to exclude npm because it returns status code 429 (too many requests) and breaks the test.


The broken link checker has two behaviors when parsing a page. For each link we check whether or not it is broken. Then we also decide whether or not to queue up that link to check its links. This second behavior is changed by the `--exclude-internal` flag. The localhost:8080/slack-invite/ link is marked as internal so we don't check to see if it is broken, but we do queue it so that we can check if it contains external links that are broken. This results in us popping off the slack domain and checking for broken external links. Now the crawler uses the current domain (slack) to decide what is an internal link. This leads to the crawler walking slack. This eventually leads to another redirect which continues for hours.

Reply:
Ok thanks for the explanation!